packages/SwingSet/src/controller.js

```diff
@@ -19,6 +19,9 @@ import { insistCapData } from './capdata';
 import { parseVatSlot } from './parseVatSlots';
 import { buildStorageInMemory } from './hostStorage';
 
+const ADMIN_DEVICE_PATH = require.resolve('./kernel/vatAdmin/vatAdmin-src');
```

Comment:
darn. More use of `require.resolve`. (ambient authority grumble... and doesn't work on xs)

I wish I had taken a look sooner.